### DISCUSSION TITLE
蓝灯团队已经在修复问题当中. 请各位耐心等待. 谢谢!

### DISCUSSION THREAD

**Discussion by MaxMKMax** [getlantern/lantern-forum-cn#325] [Announcements]
你们的声音我们都听到了. 请给我们一点时间. 非常抱歉影响了体验感. 也谢谢你们支持蓝灯! 

如果各位方便的话. 帮忙提交应用内报告. 然后把 8 位数票号给我们. 这会帮很大的忙.

打个比方:
票号 (Ticket number/ Log number) - 12345678
系统- Window 10, 11 / IOS xxx/ Mac OS xxx/ Android xxx?
版本- 7.8.3 / x.x.xxx

**Comment by Aiden-Yu** [comment]
我的 MacOS 14.4.1 M2，一顿操作连是能连上，但网速慢，急需的朋友可以试试。我先是重装各种老版本，装到蓝灯v6.10.5不闪退，但授权专业版失败（503），我就想按@lishujun说的删除配置文件试了试：
卸载蓝灯，删除 ~/.lanternsecrets，重装蓝灯v7.8.3不闪退了，重新授权专业版成功，但会看到提示“蓝灯无法设置系统代理”，蓝灯图标有个红色❗️，重启系统就可以了。就是分配的新加坡路线速度很慢，刷google网页还行，youtube看视频就比较难了。
